src/PaymentGateways/Actions/RegisterPaymentGateways.php

```diff
@@ -36,8 +38,8 @@ class RegisterPaymentGateways
      */
     public $gateways = [
         // When complete, the Test Gateway will eventually replace The legacy Manual Gateway.
-        //TestGateway::class,
-        //TestGatewayOffsite::class,
+        TestGateway::class,
+        TestGatewayOffsite::class,
```

Comment:
I'm assuming leaving these open was not intentional but I want to point out that I don't think we're quite ready to fully replace the legacy "Test Donation".  We're really close though and I'm thinking we can open a PR for that specific purpose very soon - just don't want customers to be confused about multiple test gateways.

Reply:
Oops! Yes, that was an oops. Resolved in f480a9a29